### PR TITLE
Mozilla Survey on 20210226

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

Survey all Mozilla libraries, runtime, and Web applications. Drop unused MozJS runtime.

Package(s) Affected
-------------------

New Packages

- `cbindgen` v0.17.0
- `erlang-22` v22.3.4.11

Updated Packages

- `0ad` v0.2.24b
- `couchdb` v3.1.1
- `firefox` v86.0
- `js-78` v78.8.0
- `nss` v3.62
- `rebar` v3.14.4
- `thunderbird` v78.8.0

Rebuilt Packages

- `cjs` v4.8.1-1
- `erlang` v23.1-1
- `js` v1.8.5-7
- `js-68` v68.12.0-4

Dropped Packages

- `coin`, lack of maintenance, also unneeded.
- `js-24`, `js-38`, `js-52`, `js-60`, no longer needed.

Security Update?
----------------

Yes.

- Firefox 86.0, [MSFA2021-07](https://www.mozilla.org/en-US/security/advisories/mfsa2021-07/).
- Thunderbird 78.8.0, [MSFA2021-09](https://www.mozilla.org/en-US/security/advisories/mfsa2021-09/).
- MozJS 78.8.0 [MSFA2021-08](https://www.mozilla.org/en-US/security/advisories/mfsa2021-09/).

Build Order
-----------

```
js erlang erlang-22 rebar js-68 couchdb js-78 cjs 0ad nss cbindgen firefox thunderbird
```

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Review by liushuyu — APPROVED** [review]

File: extra-games/0ad/autobuild/build [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
+    make
+)
+
+abinfo "Creating installation directories ..."
+install -dv "$PKGDIR"/usr/{bin,lib/0ad,share/0ad/data}
+
+abinfo "Installing binaries ..."
+install -Dm755 "$SRCDIR"/binaries/system/pyrogenesis \
+    "$PKGDIR"/usr/bin
+install -Dm755 "$SRCDIR"/binaries/system/*.so \
```
Comment (OriginCode) [inline extra-games/0ad/autobuild/build]: Need verbosity.

**Review by OriginCode — CHANGES_REQUESTED** [review]

**Review by OriginCode — APPROVED** [review]

**Comment by szclsya** [comment]
firefox 86.0 LGTM on AMD64.

**Comment by OriginCode** [comment]
firefox and thunderbird LGTM on AMD64.

**Comment by MingcongBai** [comment]
Firefox and Thunderbird FTBFS on Loongson 3. Skipping now in interest to ship security update. cc @FlyGoat

**Comment by MingcongBai** [comment]
Firefox 86 also comes with stability improvement for POWER, tabs no longer crash when attempting to play videos.

**Comment by MingcongBai** [comment]
@l2dy Please assign an AOSA.

**Comment by l2dy** [comment]
Use AOSA-2021-0007.

**Pull request merged**